ext/nnstreamer/tensor_filter/tensor_filter_caffe2.cc

```diff
@@ -90,7 +90,6 @@ std::map <char*, Tensor*> Caffe2Core::inputTensorMap;
  */
 Caffe2Core::Caffe2Core (const char * _model_path, const char *_model_path_sub)
 {
-  g_assert (_model_path != NULL && _model_path_sub != NULL);
   init_model_path = g_strdup (_model_path);
```

Comment:
What happens if they are NULL?
Do we handle such conditions properly at the start of a pipeline?

Reply:
[Line 465](https://github.com/nnsuite/nnstreamer/blob/37b0603e9826d9328f50f6a3c1d4bdd645c2c06f/ext/nnstreamer/tensor_filter/tensor_filter_caffe2.cc#L465) in this file ensures that they both are not NULL.

Reply:
Because constructor cannot return with failure, argument checks are performed in caller function.